framework/UPGRADE.md

```diff
@@ -8,6 +8,16 @@ if you want to upgrade from version A to version C and there is
 version B between A and C, you need to following the instructions
 for both A and B.
 
+
+Upgrade from Yii 2.0.4
+----------------------
+  
+* The signature of several `saveMessagesTo...` methods in `yii\console\controllers\MessageController` is changed. They now have an additional parameter named `markUnused`:
```

Comment:
Should be:
```
The signature of the following methods in `yii\console\controllers\MessageController` has changed:
```
Better will be following `yii\base\View::beforeRender()` and `yii\base\View::afterRender()` note at:
https://github.com/yiisoft/yii2/blob/master/framework/UPGRADE.md#upgrade-from-yii-20-rc